JULIERAJ/KIDS-FIRST-v2
# Refactor/109/use identifiers instead of index

**What has been done**
- Replaced the usage of indexes with unique identifiers in various components.

- Updated components:
                -    src/components/Calendar/KFToolbar.jsx
                -    src/components/Calendar/events.js
                -    src/components/EventModal/EventModal.jsx
                -    src/components/Footer/Footer.jsx
                -    src/components/HomeDashboard/WeeksOverview.jsx
 
**Testing Instructions**

WeeksOverview:

           -    Navigate to the dashboard/homedashboard
          -     Verify that the overview of this week's event display the correct days and events
          
![image](https://github.com/JULIERAJ/KIDS-FIRST-v2/assets/86498834/8620eea5-814a-4e00-8628-213d4ad7846f)

EventModal:
          -     Navigate to the dashboard/calendarview
          -     Click Create button
          -     Verify that the 'Share with' and 'Repeat' fields display the correct options.
![image](https://github.com/JULIERAJ/KIDS-FIRST-v2/assets/86498834/07d2a020-39e4-4cff-9f88-15dc2740a621)
![image](https://github.com/JULIERAJ/KIDS-FIRST-v2/assets/86498834/b5565a40-01a6-4e28-a163-c84737f3f171)


KFToolbar:
          -     Navigate to the dashboard/calendarview
          -     Verify that the kids' buttons are displayed correctly. 
          
![image](https://github.com/JULIERAJ/KIDS-FIRST-v2/assets/86498834/a1368548-a926-43e6-a246-b1d039fd62a4)


Footer: 
          -     The code has been refactored, but the component does not seem to be imported for testing.
